README.md

```diff
@@ -1,5 +1,61 @@
-# curriculum-cr-training-contact-form
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This repo host an initial code for the tasks performed during the Code Reviewers training.
+# Portfolio
 
-Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+This project is part of the Microverse curriculum for software development and it's meant to be a template for our own portfolio later on.
+Here you will find some projects that I've been doing in the past as well as a contact form, an about section to get to know me a bit better and a list of my skills. Make sure to check it out!
+
+## Built With
+
+- HTML, CSS, JavaScript
+- Linters used: ESLint, Stylelint and webhint.
+- No frameworks, just plain JavaScript.
+
+## Getting Started
+
+**To get started with this project, all you have to do is cloning it!**
+
+If you want to see this repo in your local machine, you have two option:
+1. You can clone it to your computer and run it with live server or a localhost.
+2. You can check at this link: [Portfolio](https://thisisjustafakelink.com)
```

Comment:
- [ ]  As the previous reviewer mentioned it's important to make sure there are no broken links in your README file. Please let's fix that  :+1: